src/System.Runtime.Serialization.Json/tests/Performance/DcjsDeserializationTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Xunit.Performance;
+using SerializationTypes;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.Json;
+using System.Xml;
+
+namespace System.Runtime.Tests.Performance
+{
+    public class DcjsDeserializationTests
+    {
+        public void RunDcjsDeserializationTest(object obj, int iterations)
+        {
+            var serializer = new DataContractJsonSerializer(obj.GetType());
+
+            using (var stream = new MemoryStream())
+            {
+                serializer.WriteObject(stream, obj);
+                foreach (var iteration in Benchmark.Iterations)
+                {
+                    using (iteration.StartMeasurement())
+                    {
+                        for (int i = 0; i < iterations; i++)
+                        {
+                            stream.Position = 0;
+                            serializer.ReadObject(stream);
+                        }
+                    }
+                }
+            }
+        }
+
+        public static byte[] CreateByteArray(int size)
+        {
+            byte[] obj = new byte[size];
+            for (int i = 0; i < obj.Length; ++i)
+            {
+                obj[i] = (byte)(i % 256);
+            }
+            return obj;
+        }
+
+        [Benchmark]
+        public void DeserializeByteArrayOf1K()
+        {
+            byte[] byteArrayOf1K = CreateByteArray(1024);
+            RunDcjsDeserializationTest(byteArrayOf1K, 10000);
```

Comment:
I think that would work for normal tests but probably not suited for perf tests.